test/unmodifiable_collection_test.dart

```diff
@@ -311,8 +311,8 @@ void testReadList(List original, List wrapped, String name) {
   });
 }
 
-void testNoWriteList(List original, List wrapped, String name) {
-  List copy = new List.from(original);
+void testNoWriteList(List<int> original, List<int> wrapped, String name) {
+  var copy = new List<int>.from(original);
```

Comment:
Can you use `.of()` here and below to avoid the explicit annotation?

Reply:
Seems so. Good catch.